.github/pull_request_template.md

```diff
@@ -1,17 +1,20 @@
-## What is this PR doing?
+# What is this PR doing?
 
 
-## How can these changes be manually tested?
+# How can these changes be manually tested?
 
 
-## Does this PR resolve or contribute to any issues?
+# Does this PR resolve or contribute to any issues?
 
 
-## Checklist
+# Checklist
 - [ ] I have manually tested these changes
 - [ ] Post a link to the PR in the group chat
 
-## Guidelines
+# Guidelines
+- Please be careful when handling disagreements and stay mindful of others
 - If your PR is not ready, mark it as a draft
+- Prefer to seek followup instead of requiring changes, especially if the PR is already a clear net improvement
+- If appropriate, consider implementing followup changes yourself instead of waiting for them from the author and open a new PR against this one.
 - The `resolve conversation` button is for reviewers, not authors
   - (But add a 'done' comment or similar)
```

Comment:
I wanna suggest adding another point -

- Keep the diff of the PR as small as possible, if the PR changes more than around 10 files, further break down the scope of the PR

The PR should only be of a limited scope, if the PR modifies 10+ files it would be better to rethink the scope of the PR and create additional issues and related PRs. (with exceptions ofcourse)
IMHO, it becomes extremely difficult and cumbersome to review a PR which changes 40+ files.